packages/components/src/internal/testHelpers.tsx

```diff
@@ -118,17 +122,20 @@ export const AppContextTestProvider: FC<AppContextTestProviderProps> = props =>
  * the returned mounted component will still be the component under test (as opposed to <AppContextProvider/>).
  * @param appContext The app context to be provided by the wrapping <AppContextProvider/>.
  * @param serverContext The server context to be provided by the wrapping <ServerContextProvider/>.
+ * @param notificationContext The notification context to be provided by the wrapping <NotificationProvider/>.
  * @param options Pass through for mount's rendering options.
+ * @param printLabelsContext The server context to be provided by the wrapping <PrintLabelsContext/>.
  */
 export const mountWithAppServerContextOptions = (
     appContext?: Partial<AppContext>,
     serverContext?: Partial<ServerContext>,
     notificationContext?: Partial<NotificationsContextState>,
-    options?: MountRendererProps
+    options?: MountRendererProps,
+    printLabelsContext?: Partial<LabelPrintingProviderProps>,
```

Comment:
should the `printLabelsContext` param go before `options`? or are you worries about breaking existing usages?

Reply:
the latter